.gitignore

```diff
@@ -16,6 +16,7 @@ local.properties
 
 # Android Studio / IntelliJ IDEA
 *.iws
+/.idea
```

Comment:
your files are still under here.

So, I guess the changes you have made are not working correctly. and also the changes are already there to ignore these folders. Please check once again before pushing.

Reply:
```
public abstract static class SingletonC implements MyApplication_GeneratedInjector,
                         ^
      com.hadiyarajesh.notex.reminder.notification.NotificationHelper is injected at
```